server/lib/data/convert-reference-data-csv-to-json.js

```diff
@@ -57,7 +57,7 @@ module.exports = async (readableStream, type, db) => {
         if (valid) {
           data = temp
             .filter(x => x.orgCode !== 'UFD')
-            .concat({ orgName: 'Undefined', orgCode: 'UFD', active: true, core: false })
+            .concat({ orgName: 'Undefined', orgCode: 'UFD', active: true, core: true })
```

Comment:
Undefined is categorised as 'core' org as it is used to map users to from the core data set of AAD. It shouldn't have been set to non-core.